src/CMakeLists.txt

```diff
@@ -257,80 +251,80 @@ if(WITH_LZ4_EXT)
   target_link_libraries(rdkafka PUBLIC LZ4::LZ4)
 endif()
 
-# Set up path to these sources for other sub-projects (tests, examples)
-# to be able to reach them.
-#set(rdkafka_SRC_DIR ${PROJECT_SOURCE_DIR}
-#    CACHE INTERNAL "${PROJECT_NAME} source dir" FORCE)
+if(MINGW)
+  message(STATUS "Detected MinGW build")
+  # MinGW ignores `#pragma comment(lib, ...)` so we need to specify link libs
+  target_link_libraries(rdkafka PUBLIC crypt32 ws2_32 secur32)
+endif(MINGW)
```

Comment:
I would just link crypt32 ws2_32 secur32 if WIN32, to make CMakeLists neat. By the way I think that crypt32 is not needed if WITH_SSL is OFF.